js/component-sections.js

```diff
@@ -1,2 +1,39 @@
 // export as global since we're not using a module system
-window.Sections = { template: '<div>Sections {{$route.params.course}} -> {{$route.params.ta}}</div>' }
+window.Sections = Vue.component('Sections',{
+  data(){
```

Comment:
Please format your code with the prettier.js code formatter and set indentation to 4 spaces